components/eam/bld/namelist_files/namelist_defaults_eam.xml

```diff
@@ -241,6 +241,14 @@
 <mam_ncl  rad="rrtmgp">atm/cam/physprops/ssam_rrtmg_c100508.nc</mam_ncl>
 <mam_nh4  rad="rrtmgp">atm/cam/physprops/sulfate_rrtmg_c080918.nc</mam_nh4>
 
+<!-- hybrown, optics for tagged species -->
+<mam_so401  rad="rrtmg">atm/cam/physprops/sulfate_rrtmg_c080918.nc</mam_so401>
+<mam_so402  rad="rrtmg">atm/cam/physprops/sulfate_rrtmg_c080918.nc</mam_so402>
+<mam_so403  rad="rrtmg">atm/cam/physprops/sulfate_rrtmg_c080918.nc</mam_so403>
+<mam_so401  rad="rrtmgp">atm/cam/physprops/sulfate_rrtmg_c080918.nc</mam_so401>
+<mam_so402  rad="rrtmgp">atm/cam/physprops/sulfate_rrtmg_c080918.nc</mam_so402>
+<mam_so403  rad="rrtmgp">atm/cam/physprops/sulfate_rrtmg_c080918.nc</mam_so403>
+<mam_soa01  rad="rrtmg">atm/cam/physprops/ocphi_rrtmg_c100508.nc</mam_soa01>
```

Comment:
Should one of these (ocphi) be added for rrtmgp too?

Reply:
No, that was an artifact from testing out other tagged species. The namelist_defaults_eam.xml changes are less important since we are specifying everything in the runscript user_nl_eam, which supersedes these modifications. I think I was changing them just to be thorough.